packages/Webkul/Shop/src/Http/Controllers/Customer/RegistrationController.php

```diff
@@ -106,13 +107,16 @@ public function store(RegistrationRequest $registrationRequest)
     public function verifyAccount($token)
     {
         $customer = $this->customerRepository->findOneByField('token', $token);
-
         if ($customer) {
             $this->customerRepository->update([
                 'is_verified' => 1,
                 'token'       => null,
             ], $customer->id);
-
+            try {
+                Mail::queue(new RegistrationNotification($customer,'customer'));
+            } catch (\Exception $e) {
+                report($e);
+            }
```

Comment:
Please dispatch the event for the sending email, as we have implement in other places.